modules/document_repository/templates/menu_document_repository.tpl

```diff
@@ -316,9 +316,13 @@
                             <label class="col-xs-4" for="site">Site<font color="red"><sup> *</sup></font></label>
                             <div class="col-xs-8">
                                 <select name="site" id = "site" class = "form-fields form-control input-sm">
-                                <option value=" "> </option>
+                                <option value=""> </option>
                                     {foreach from = $Sites item=val key=k}
-                                        <option value={$k}>{$val}</option>
+                                        {if $val == "Any"}
+                                            <option value="0">Any</option>
```

Comment:
@kongtiaowang If you do this, the value 0 gets inserted in the database for the site but 0 is not a site. instead the empty string should be handled